content/en/docs/tasks/security/proxy-cert/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Provision a certificate through a sidecar [Experimental]
+description: Provision a certificate through a sidecar.
+weight: 100
+keywords: [certificate,sidecar]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Istio sidecars can obtain certificates through
+the secret discovery service.
+For an application without a sidecar to get a certificate,
+it may deploy a sidecar to generate a certificate through
+the secret discovery service and share the certificate with the application
+through a shared memory.
+This task uses Prometheus as an example application to show provisioning
+a certificate through such mechanism.
+
+## Getting started
+
+Install Istio:
+
+{{< text bash >}}
+$ istioctl manifest apply
+{{< /text >}}
+
+## Provision a key and certificate for an application
```

Comment:
Deliver key and certificate to application via files

Reply:
Done.